app/database/operator/server_data_operator/transformers/reaction.test.ts

```diff
@@ -23,8 +23,6 @@ describe('*** REACTION Prepare Records Test ***', () => {
                     post_id: 'ps81iqbddesfby8jayz7owg4yypoo',
                     emoji_name: 'thumbsup',
                     create_at: 1596032651748,
-                    update_at: 1608253011321,
-                    delete_at: 0,
```

Comment:
Why these are removed?

Reply:
This raw is supposed to be a `Reaction` type, which has no update at or delete at defined.